src/lib/y2storage/subvol_specification.rb

```diff
@@ -244,5 +245,28 @@ def self.fallback_list
       subvols.each { |subvol| subvol.archs = SUBVOL_ARCHS[subvol.path] }
       subvols.sort!
     end
+
+    # Reads the <subvolumes> section of control.xml, creating a fallback list if
+    # nothing is specified.
+    #
+    # @note Only specs that make sense for the current architecture are returned.
+    #
+    # @return [Array<SubvolSpecification>]
+    def self.for_current_product
```

Comment:
More duplication of responsibilities (filtering by product) that should be killed by killing `Planned::BtrfsSubvolume`